dataset-snippets/measurement_by_age_and_sex_at_birth.ggplot

```diff
@@ -7,20 +7,20 @@
 options(repr.plot.height = 16, repr.plot.width = 16)
 
 # There could be many different measurements in the dataframe. By default, plot the first one.
-measurement_to_plot <- unique(measurement_df$STANDARD_CONCEPT_NAME)[1]
+measurement_to_plot <- unique(measurement_df$standard_concept_name)[1]
```

Comment:
Hm.. did column name casing change across dataset versions? I missed this if so

Reply:
Apparently so! 

I cloned an older testing workspace and then exported the dataset to a notebook and noticed all the dataset builder snippets were broken due to the casing change. You can see the dataset builder export in [this workspace](https://workbench.researchallofus.org/workspaces/aou-rw-d03ec71b/duplicateoftestanddevelopcodesnippetsonaouv5/notebooks).